client/components/payment-method-fees-pill/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Pill from '../pill';
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
Using this rule because in the end we will use the `id` of the payment method to gather its fees